.github/workflows/ruff.yml

```diff
@@ -0,0 +1,9 @@
+name: Ruff
+on: [push, pull_request]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: chartboost/ruff-action@v1
+        continue-on-error: true
```

Comment:
Yup, this is just for information for now. We need to remember to remove it (assuming its default is `false`) or set to `false` when we want it to stop.